app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @review = Review.new(params_review)
```

Comment:
While the way you are building Review isn't bad. You can do it in a little bit different way.

``` ruby
@video = Video.find(params[:video_id])
@review = @video.reviews.build(review_params.merge!(user: current_user))
```

You are searching for `@video` below so you can just use it to build Review as well.